docs/architecture-decisions/.template.md

```diff
@@ -0,0 +1,70 @@
+---
```

Comment:
Will there be a guideline for where an ADR is required? (e.g. API change, major refactor, XL feature, deprecation etc)